content/consulting/software-audit.mdx

```diff
@@ -1,20 +1,20 @@
 ---
 seo:
-  title: " Software Auditing"
+  title: ' Software Auditing'
```

Comment:
Why is there a space at the beginning?

Reply:
I think I accidentally added it, just removed it.